shepherd/Goal.py

```diff
@@ -67,4 +78,50 @@ def apply_powerup(self, effect, alliance):
             effect - enum representing an effect to be applied to the goal
             alliance - the alliance trying to apply the effect to the goal
         """
-        pass
+        if effect == POWERUP_TYPES.TWO_X:
+            if alliance.name == CONSTANTS.BLUE:
+                if self.blue_two_x_timer.is_running():
+                    return -1
```

Comment:
Not really a part of this pull request but is there a reason that we are returning integers defined in the above docstring instead of potential global constants. I think changing this would help with the readability of this code and would make the code that calls this method more clear as well.